src/actions/dot/sudoAs.ts

```diff
@@ -45,7 +49,9 @@ export const sudoAs = async (cmd: Command) => {
     .readFileSync(prevHashes, { encoding: "utf-8" })
     .split("\n");
 
-  const api = await initApi(wsEndpoint, types);
+  // const api = await initApi(wsEndpoint, types);
```

Comment:
```suggestion
  // const api = await initApi(wsEndpoint, types);
```

shouldn't this line and the one below it be equal if `types` is `null`?